combat.c

```diff
@@ -1 +1,143 @@
-// Maybe write something like initCombat(int location) here
\ No newline at end of file
+#include <stdio.h>
+#include <stdlib.h>
+#include "game_object.h"
+typedef struct {
+    int health;
+    int maxHealth;
+    int damage;
+    char name[50];
+} Enemy;
+void createcombat(Game* game) {
+    printf("\n=== COMBAT START ===\n");
+    int location = game->level;
+    int numEnemies = 1 + (location / 3); //enemies based on us to determand their level 
```

Comment:
hardcoding here should not be allowed, what if we have multiple newbie friendly zone. Monster should use a fixed stat then multiply it with local difficulty